1-WebApp-OIDC/1-5-B2C/appsettings.json

```diff
@@ -1,16 +1,17 @@
 ﻿{
-	"AzureAdB2C": {
-		"Instance": "https://<your-tenant-name>.b2clogin.com",
-		"ClientId": "<web-app-application-id>",
-		"Domain": "<your-b2c-domain>",
-		"CallbackPath": "/signin/B2C_1_sign_up_in",
-		"SignedOutCallbackPath": "/signout/B2C_1_sign_up_in",
-		"SignUpSignInPolicyId": "<your-sign-up-in-policy>"
-	},
-	"Logging": {
-		"LogLevel": {
-			"Default": "Warning"
-		}
-	},
-	"AllowedHosts": "*"
+  "AzureAdB2C": {
+    "Instance": "https://fabrikamb2c.b2clogin.com",
+    "ClientId": "90c0fe63-bcf2-44d5-8fb7-b8bbc0b29dc6",
+    "Domain": "fabrikamb2c.onmicrosoft.com",
+    "SignedOutCallbackPath": "/signout/B2C_1_susi",
+    "SignUpSignInPolicyId": "B2C_1_susi",
+    //"CallbackPath": "/signin/B2C_1_sign_up_in"  // defaults to /signin-oidc
```

Comment:
do we want to remove the line altogether?

Reply:
It's still mentioned in the readme